### PR TITLE
feature - take line follower out

### PR DESCRIPTION
**Pull request by RobertLucian**
Remove the line follower from the GoPiGo and also adjusts the paths accordingly. Tested with https://github.com/DexterInd/DI_Sensors/pull/46 and it works fine.

Pay real attention to the paths as those have been a big pain so far! One bad path and we end with something non functional.

This goes hand in hand with https://github.com/DexterInd/DI_Sensors/pull/46.

### CODE REVIEW

File: Setup/install.sh [hunk 1]
```diff
@@ -283,8 +244,6 @@ install_wiringpi
 install_spi_i2c
 # no longer installing avr for arduino
 # install_avr
-install_line_follower
-install_control_panel
```
Comment (CleoQc) [inline Setup/install.sh]: why is the control panel removed?

File: Software/Python/easygopigo.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def _ifMutexRelease(mutex_enabled = False):
     is_line_follower_accessible = True
 except:
     try:
-        sys.path.insert(0, '/home/pi/GoPiGo/Software/Python/line_follower')
+        sys.path.insert(0, '/home/pi/Dexter/DI_Sensors/Python/di_sensors/red_line_follower/line_follower')
```
Comment (CleoQc) [inline Software/Python/easygopigo.py]: if line_follower is a system-wide package, then load it via an import. 
This was a hack for when it wasn't a system-wide package.

File: Software/Scratch/GoPiGoScratch.py [hunk 3]
```diff
@@ -373,25 +373,25 @@ def run(self):
...
+			if en_gpg:
 				s.sensorupdate({'ir':a})
-				
+
 		# Get the value from the Dexter Industries line sensor
 		elif msg.lower()=="LINE".lower():
 			try:
 				import sys
-				sys.path.insert(0, '/home/pi/Dexter/GoPiGo/Software/Python/line_follower')
+				sys.path.insert(0, '/home/pi/Dexter/DI_Sensors/Python/di_sensors/red_line_follower/line_follower')
```
Comment (CleoQc) [inline Software/Scratch/GoPiGoScratch.py]: again, feel free to load it as a proper package.

File: Setup/install.sh (hunk 1 above)
Comment (RobertLucian) [reply Setup/install.sh]: Good call. I knew something was missing. Thanks.

File: Software/Python/easygopigo.py (hunk 2 above)
Comment (RobertLucian) [reply Software/Python/easygopigo.py]: Okay. Then this second `try/except` block is going out, as well as the other one you mentioned.

File: Software/Scratch/GoPiGoScratch.py (hunk 3 above)
Comment (RobertLucian) [reply Software/Scratch/GoPiGoScratch.py]: Right.

**Pull request merged**